### PR TITLE
Enhance preview chat #1

### LINKED ISSUE #2686: Enhance preview chat
**Actual Behaviour**

Currently, only the bubble is shown

**Expected Behaviour**

Enhance current chat preview

Options:

1. Play Youtube video
2. Play SoundCloud song
3. Show me an image
4. Tell me a joke

Simple chat now, and basic 3-4 chat select options user can see:

Design Idea 1:
Screenshot 2019-07-27 at 11 24 02 PM
Screenshot 2019-07-27 at 11 24 30 PM

Idea 2, For first time user:

Screenshot 2019-07-27 at 11 27 59 PM

We can show the top used skills:

Screenshot 2019-07-27 at 11 31 59 PM

Idea 3:

Screenshot 2019-07-27 at 11 34 17 PM
Screenshot 2019-07-27 at 11 34 49 PM

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2686, #2700 

Changes: 
- Changed design of chat preview
- Change Message search Design

Demo Link: https://pr-2699-fossasia-susi-web-chat.surge.sh

Screenshots of the change:

Screenshot 2019-07-28 at 7 22 40 PM

Screenshot 2019-07-28 at 7 22 48 PM

### CODE REVIEW

File: src/components/ChatApp/SearchField.react.js [hunk 1]
```diff
@@ -1,19 +1,44 @@
...
+import _CloseIcon from '@material-ui/icons/Close';
+import _UpIcon from '@material-ui/icons/ExpandLess';
+import _DownIcon from '@material-ui/icons/ExpandMore';
 
 const ESCAPE_KEY = 27;
 const F_KEY = 70;
 
+const NavigationIconStyle = css`
+  width: 0.8rem;
+  height: 0.8rem;
```
Comment (harsh-jindal) [inline src/components/ChatApp/SearchField.react.js]: Change the width and height to 1rem. Currently it looks too small.
Rest LGTM 🚀

Comment (fragm3) [reply src/components/ChatApp/SearchField.react.js]: Let's wait for @akshatnitd review as well

**Comment by fragm3** [comment]
@akshatnitd made the changes:
Screenshot 2019-07-28 at 9 40 03 PM

**Comment by mariobehling** [comment]
The chat box does not close anymore from the top menu chat icon now. Also the rounded corners do not correspond to the rounded corners of other elements on the site. The corners should have the same rounding as the blue buttons for example.

**Pull request merged**